dell/latitude/5520/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  # Essential Firmware
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  # Cooling Management
+  services.thermald.enable = lib.mkDefault true;
+
+  # Video Drivers (Intel Iris Xe)
+  services.xserver.videoDrivers = lib.mkDefault [ "intel" ];
```

Comment:
So the `intel` is better here then using the  Kernel modsettings that we do on other platforms?

Reply:
Maybe make a longer comment describing why this is the case since it's different from what we normally do.

Reply:
I have no idea which is better. But this is how it is done for the related [Dell Latitude 3480](https://github.com/NixOS/nixos-hardware/commit/d7d897fe2499df6a4ea561d7ed642d1c1f30c98e), so I thought that is is supposed to be that way. If this is not the default, I will change that.

Reply:
Maybe just drop the line and it will use the builtin kms driver. If you experience any issues that are fixed by this option. We can still add it back.